docs/src/components/landing/SignUp.js

```diff
@@ -0,0 +1,160 @@
+        address so that we can keep you up to date.
+      </Alert>
+    );
+  }
+  return (
+    <Form onSubmit={handleSubmit} sx={sx}>
+      <Box
+        sx={{
+          display: 'flex',
+          gap: 0,
+          width: { xs: '100%', sm: 'auto' },
+          maxWidth: 320,
+        }}
+      >
+        <InputBase
+          id="email-landing"
+          name="email"
+          type="email"
+          placeholder="example@email.com"
+          inputProps={{ required: true }}
+          value={form.email}
+          onChange={(event) => setForm({ email: event.target.value, status: 'initial' })}
+          sx={{
+            mr: { xs: 0, sm: 1 },
+            mb: { xs: 1, sm: 0 },
+            minWidth: { xs: 220, sm: 360 },
+            width: { xs: '100%', sm: 'auto' },
+            bgcolor: (theme) =>
+              theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : '#fff',
+            boxShadow: (theme) =>
+              theme.palette.mode === 'dark'
+                ? '0 1px 2px 0 rgba(0 0 0 / 1)'
+                : '0 1px 2px 0 rgba(0 0 0 / 0.1)',
+            borderRadius: 1,
+            border: '1px solid',
+            borderColor: (theme) =>
+              theme.palette.mode === 'dark' ? theme.palette.primaryDark[500] : 'grey.400',
+            px: 1,
+            py: 0.5,
+            height: 50,
+            typography: 'body2',
+            '&:hover': {
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark' ? theme.palette.primaryDark[300] : 'grey.400',
+              boxShadow: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? '0 1px 2px 0 rgba(0 0 0 / 1)'
+                  : '0 1px 2px 0 rgba(0 0 0 / 0.2)',
+            },
+            [`&.${inputBaseClasses.focused}`]: {
+              borderColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[300]
+                  : theme.palette.primary[500],
+              outline: '3px solid',
+              outlineColor: (theme) =>
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[500]
+                  : theme.palette.primary[300],
+            },
+          }}
+        />
+        <Button
+          disabled={form.status === 'loading'}
+          type="submit"
+          size="large"
+          variant="contained"
+          endIcon={<KeyboardArrowRightRounded />}
+          sx={{ width: { xs: '100%', sm: 'auto' } }}
+        >
+          Sign up
+        </Button>
+      </Box>
+      {form.status === 'failure' ? (
+        <FormHelperText
+          sx={{ color: (theme) => (theme.palette.mode === 'dark' ? 'warning.500' : 'warning.800') }}
+        >
+          Oops! something went wrong, please try again later.
+        </FormHelperText>
+      ) : null}
+    </Form>
+  );
+};
+
+SignUp.propTypes = {
+  sx: PropTypes.object,
+};
+
+export default SignUp;
```

Comment:
For the next components, I think that we will be better off writing them in TypeScript.